aodndata/gsla/handler.py

```diff
@@ -78,10 +88,14 @@ def preprocess(self):
                 )
 
             netcdf_file = netcdf_collection[0]
+            # setting the path of the gz file with the gunqipped file
+            netcdf_file_gz.dest_path = self.dest_path(netcdf_file.src_path)
             # Nothing to do with *.nc. Talend can harvest *.nc.gz. Set to NO_ACTION
             netcdf_file.publish_type = PipelineFilePublishType.NO_ACTION
 
-            result_previous_version_creation_date = self.get_previous_version_creation_date(netcdf_file.name)
+            # we don't know the product type (DM00 or DM01) of the file already on s3 in order to deduce its path. So we
+            # to get the product type from the file in incoming
```

Comment:
Fix sentence in comment. Missing "need" somewhere?

Reply:
done